app/screens/channel_add_members/channel_add_members.tsx

```diff
@@ -258,6 +268,27 @@ export default function ChannelAddMembers({
             edges={['top', 'left', 'right']}
             nativeID={SecurityManager.getShieldScreenId(componentId)}
         >
+            {showBanner && (
+                <SectionNotice
+                    type='info'
+                    title={formatMessage({
+                        id: 'channel.abac_policy_enforced.title',
+                        defaultMessage: 'Channel access is restricted by user attributes',
+                    })}
+                    text={formatMessage({
+                        id: 'channel.abac_policy_enforced.description',
+                        defaultMessage: 'Only people who match the specified access rules can be selected and added to this channel.',
+                    })}
+                    tags={attributeTags.length > 0 ? attributeTags : undefined}
+                    isDismissable={true}
+                    onDismissClick={useCallback(() => setShowBanner(false), [])}
```

Comment:
My friend move this to a function on the component

Reply:
ok, done in my last commit.